### ISSUE TITLE
Epic►012►Parent: Creating V²DIY Board 2016W23

### ISSUE THREAD

**Issue by VillageHubertChen** [V-Squared/V-Squared.github.io#117]
# Gated by Items:
-  [x] BC: Epic►011►Parent: Page Structure for V²►DIY #122 
## Unraveling too many dependencies to stop gating
- DIY Article have no dependencies to Configuration Data Structure
- DIY Articles > 50% finished
- DIY Article structure is pretty finished

**Conclusion:** We begin ASAP to create DIY Pages and move DIY pages from Wiki to Code. This way Bobby can start writing ASAP. Once done HC and LC have less pressure to finish the rest, as they are no longer gating Bobby
# Tasks
- [x] HC, LC: Create Pages according → Epic►011►Parent: Page Structure for V²►DIY #122 
- [x] HC: Begin moving Articles I wrote in the Wikis to V²DIY. 
  - [x] Begin to sort in the Articles Bobby wants to start writing on!
    - https://v-squared.github.io/diy/parts/
    - https://v-squared.github.io/diy/compare/
- [x] HC: Create → Epic►010►Parent: V²DIY User Stories / Data Structure / Apps #125
  - to reduce dependencies, we will first focus on completing DIY Articles, then go for DIY Apps
- [x] HC: Create → Epic►015►Parent: Compile Know How►Writing Articles #141
- [x] HC: Hand over baton to BC to begin writing on first Articles already moved

**Comment by VillageHubertChen** [comment]
The creation of the DIY Board is complete. Creating all its articles is by far not complete. Each Article will have its own Issue to be worked on.
